bundles/ch.elexis.core.l10n/src/ch/elexis/core/l10n/messages_en.properties

```diff
@@ -5249,6 +5275,8 @@ VerrechnungsDisplay_confirmChangeTextCaption = Really change text?
 
 VerrechnungsDisplay_doBill = offset
 
+VerrechnungsDisplay_doNotCharge = Anz
+
```

Comment:
This value, as far as i see, is not referenced within the Messages.java file. Could you try to find a means to identify such disconnected property entries, and remove them?

Reply:
Sorry no. That would be another job on it's own, because there is a lot of mess there.